### PR TITLE
Feature/Visual-Updates

### PR DESCRIPTION
**Pull request by charlie-tf**

### CODE REVIEW

File: .github/scripts/set-environment.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ elif [[ "$GITHUB_REF" == refs/heads/main ]]; then
   # TODO: use AWS code artifact repository for staging
   # For now, staging takes care of prod npm update.
   ARTIFACT_REGISTRY_PROVIDER=npm
+elif [[ "$GITHUB_REF" == refs/heads/feature/visual-updates ]]; then
+  # While this effort is in progress, we would like to publish to npm
+  # from this feature branch. The versions will be in the 2.x.x series.
+  ARTIFACT_REGISTRY_PROVIDER=npm
```
Comment (karthikbtf) [inline .github/scripts/set-environment.sh:16]: oh wait, remove all this?

File: .github/workflows/develop.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ name: Deployment
 
 on:
   push:
-    branches: [main]
+    branches: [main, feature/visual-updates]
   pull_request:
-    branches: [main]
+    branches: [main, feature/visual-updates]
```
Comment (karthikbtf) [inline .github/workflows/develop.yml:7]: undo?

File: .github/scripts/set-environment.sh (hunk 1 above)
Comment (charlie-tf) [reply .github/scripts/set-environment.sh:16]: oops good call!

File: .github/workflows/develop.yml (hunk 2 above)
Comment (charlie-tf) [reply .github/workflows/develop.yml:7]: 👍

**Review by karthikbtf — APPROVED** [review]
lgtm

**Pull request merged**